tests/Feature/Api/UploadPhotoTest.php

```diff
@@ -266,10 +266,6 @@ public function test_uploaded_photo_can_have_different_mime_types()
         // JPEG
         $imageAttributes = $this->getImageAndAttributes('jpeg');
         $this->post('/api/photos/submit', $this->getApiImageAttributes($imageAttributes))->assertOk();
-
-        // HEIC
-//        $imageAttributes = $this->getImageAndAttributes('heic');
-//        $this->post('/api/photos/submit', $this->getApiImageAttributes($imageAttributes))->assertOk();
```

Comment:
We can add these again?

Reply:
I added them and the tests were failing in the Github Action. That happens because Github servers that run our tests don't have heic support. Plus, every user that wants to contribute to OLM would have to install heic support on their local machine to run tests.

Reply:
We should delete them so, maybe add a comment that explains why heic is not tested